smokes/e2e/about.scenarios.coffee

```diff
@@ -3,14 +3,26 @@
 # to use previous and next link
 
 aboutPage = require('./pages/about.coffee')
+homePage = require('./pages/home.coffee')
+builderPage = require('./pages/builder.coffee')
 
-describe('', () ->
+describe 'about test', () ->
     about = null
+    home =  null
+    builder = null
 
-    beforeEach(() ->
+    beforeEach () ->
         about = new aboutPage('runtests')
-    )
+        builder = new builderPage('runtests', 'force')
+        home = new homePage()
+        builder.goDefault()
```

Comment:
It probably is because you need to go to the home page before searching for the login button.
with builder.goDefault() you go to the builder page, and look for the login page, which works too.